Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml

```diff
@@ -815,14 +815,16 @@
             <Class class="MaterialProfileType" />
         </Source>
         <Target multiplicity="(1..1)" roleLabel="is referenced by" polymorphic="false">
-            <Class class="MaterialProfile" />
+            <Class class="MaterialProfileDefinition" />
         </Target>
     </ECRelationshipClass>
 
-    <ECEntityClass typeName="MaterialProfileDefinition" modifier="abstract" displayLabel="Material Profile Definition" description="Base class defining an element that pairs Profiles and PhysicalMaterials.">
+    <ECEntityClass typeName="MaterialProfileDefinition" modifier="None" displayLabel="Material Profile Definition" description="Base class defining an element that pairs Profiles and PhysicalMaterials.">
```

Comment:
Was this change from "abstract" to concrete intended? It feels strange looking at the other changes in this PR (i.e. MaterialProfileDefinition continues to be used primarily as a base class).

Reply:
Thanks for noticing this.